exporter/util/epoch/parse.go

```diff
@@ -11,13 +11,19 @@ import (
 
 const (
 	frontendSourceDateEpochArg = "build-arg:SOURCE_DATE_EPOCH"
-
-	KeySourceDateEpoch = "source-date-epoch"
 )
 
-func ParseBuildArgs(opt map[string]string) (string, bool) {
```

Comment:
Saw the other comment as well, but if we remove this one, we should probably first mark it deprecated (did we search if there's other consumers of this?)

Reply:
Reverted these changes so it won't be removed now.